src/map/trade.cpp

```diff
@@ -413,10 +413,23 @@ void trade_tradeadditem(map_session_data *sd, int16 index, int16 amount)
 	// Locate a trade position
 	ARR_FIND( 0, 10, trade_i, sd->deal.item[trade_i].index == index || sd->deal.item[trade_i].amount == 0 );
 	if( trade_i == 10 ) { // No space left
-		// clif_tradeitemok(*sd, index, EXITEM_ADD_FAILED_OVERWEIGHT); // We do not know if the server respond with this or not since the official client prevents this case client-side.
```

Comment:
Could @Playtester verify this?
I would like to leave the comment at least "official client prevents this" or not.

Reply:
I already tested it yesterday on both Aegis pre-re an renewal, but I'd add the comment